File: Odd Sum Pair.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,c;
        scanf("%d%d%d",&a,&b,&c);
        if((a+b)%2!=0||(b+c)%2!=0||(a+c)%2!=0)
        printf("YES\n");
        else
        printf("NO\n");
        
    }
}

File: Odd Sum Pair.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            int c=sc.nextInt();
            if((a+b)%2==1||(b+c)%2==1||(a+c)%2==1)
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}